BUG: `babel-eslint` is not honoring code comment `global` declarations

IMPACT: Control comments like `/*global document:false*/` don't work to make eslint happy.

This just randomly happened in build: https://travis-ci.org/FormidableLabs/formidable-react-component-boilerplate/builds/71981956

for a non code change. Nuking `node_modules` locally and re-installing reproduces the error. This means likely some underlying module is not properly implementing semver / has a new bug.

```
 $ eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx

demo/app.jsx
  15:16  error  "document" is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)
```

--- a/.eslintrc-react
+++ b/.eslintrc-react
@@ -1,8 +1,3 @@
 ---
 "extends":
   - "defaults/configurations/walmart/es6-react"
-"globals":
-  # TODO: REMOVE `globals` section of this document.
-  # BUG: babel-eslint fails to honor control comment globals.
-  # github.com/FormidableLabs/formidable-react-component-boilerplate/issues/19
-  "document": false

--- a/demo/app.jsx
+++ b/demo/app.jsx
@@ -1,5 +1,3 @@
-// BUG: babel-eslint fails to honor control comment globals.
-// github.com/FormidableLabs/formidable-react-component-boilerplate/issues/19
 /*global document:false*/
 import React from "react";
 import BoilerplateComponent from "../src/index";

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "webpack": "^1.10.0"
   },
   "devDependencies": {
-    "babel-eslint": "^3.1.0",
+    "babel-eslint": "^3.1.25",
     "eslint": "^0.24.1",
     "eslint-config-defaults": "^3.0.3",
     "eslint-plugin-react": "^2.6.4",

